spec/system/publishers_can_copy_a_vacancy_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe "Copying a vacancy" do
   let(:publisher) { create(:publisher) }
-  let(:school) { create(:school) }
+  let(:school) { create(:school, safeguarding_information: nil) }
```

Comment:
I assume you tried to set `safeguarding_information` in the factories and the trouble it caused outweighed the awkwardness of doing this everywhere?